src/openms/source/CHEMISTRY/EnzymesDB.cpp

```diff
@@ -53,12 +53,11 @@ namespace OpenMS
   EnzymesDB::EnzymesDB()
   {
     readEnzymesFromFile_("CHEMISTRY/Enzymes.xml");
```

Comment:
This is not a valid path in most cases (depending on the current working directory)!
It leads to crashes with the following error:
> Error: File not found (the file 'CHEMISTRY/Enzymes.xml' could not be found)